LDAP/ldap2s3/ldap2s3.py

```diff
@@ -96,7 +110,7 @@ def normalize_ssh(self, sshlist):
         ret = []
         for s in sshlist:
             ss = s.decode('utf-8')
-            ret.append(ss.strip(' '))
+            ret.append(ss.strip(' ').strip('\r\n'))
```

Comment:
I added this just to make it nice here as i noticed they sometimes have these CRLFs...